server/lib/adapters/mysql.js

```diff
@@ -74,13 +75,13 @@ export function runQuery(client, query, options = {}) {
       }
 
       // Run the query.
-      return client.query(params, (queryError, rows) => {
+      return client.query(params, (queryError, rows, fieldPackets) => {
         client.end();
```

Comment:
Hmm I have a question about this line because I am getting error every time i run query: 
`Error: Cannot enqueue Quit after invoking quit.`
I think this is caused by duplicated client.end() invocation. Here is first and the second one is in sync-agent when we are getting result: 
https://github.com/hull-ships/hull-sql/blob/develop/server/lib/sync-agent.js#L134
https://github.com/hull-ships/hull-sql/blob/develop/server/lib/adapters/mysql.js#L31